src/generic/grid.cpp

```diff
@@ -5889,6 +5886,69 @@ void wxGrid::RefreshBlock(int topRow, int leftCol,
     }
 }
 
+void wxGrid::RefreshArea(int areas)
+{
+    if ( areas == wxGA_All )
+    {
+        // Normally if we want to refresh the entire grid we call
+        // Refresh() instead, but to make this function consistent
+        // and correct, just forward to wxScrolledCanvas::Refresh()
+        // if the argument is wxGA_All.
+        wxScrolledCanvas::Refresh();
+        return;
+    }
+
+    // Corner area
+    if ( (areas & wxGA_Corner) != 0 )
+        m_cornerLabelWin->Refresh();
+
+    int cw, ch;
+    GetClientSize(&cw, &ch);
+
+    // Cells area
+    if ( (areas & wxGA_Cells) != 0 )
+    {
+        const wxRect rect = wxRect(GetRowLabelSize(), GetColLabelSize(), cw, ch);
```

Comment:
Pretty minor, but this could be just
```suggestion
        const wxRect rect(GetRowLabelSize(), GetColLabelSize(), cw, ch);
```
(if this is changed, it should be changed below too, for consistency).